src/tests/Directory.Build.targets

```diff
@@ -74,6 +79,13 @@
     </ProjectReference>
   </ItemGroup>
 
+  <!-- Inject the XUnit wrapper generator into test build for [Fact]-style tests -->
+  <ItemGroup Condition="$(IsFactStyleTest)">
+    <ProjectReference Include="$(MSBuildThisFileDirectory)/Common/XUnitWrapperGenerator/XUnitWrapperGenerator.csproj" OutputItemType="Analyzer" ReferenceOutputAssembly="false" />
+  </ItemGroup>
+
+  <Import Project="Common/XUnitWrapperGenerator/XUnitWrapperGenerator.props" Condition="$(IsFactStyleTest)" />
+
```

Comment:
We can reference the generator and import the MSBuild file for all tests. The generator should be a no-op for non-Fact-style tests. If it isn't, I can make changes to the generator to fix that.